spaghetti/tests/test_network_from_gdf.py

```diff
@@ -42,7 +43,7 @@ def test_contiguity_weights_graph(self):
                          [(2, 2), (3, 2), (4, 45), (5, 82), (6, 48)])
 
     def test_distance_band_weights(self):
-        #I do not trust this result, test should be manually checked.
+        # I do not trust this result, test should be manually checked.
```

Comment:
ditto here?

Reply:
this and the [above](https://github.com/pysal/spaghetti/pull/60#pullrequestreview-144625090) were added before I ever became a part of `pysal`. I will look into removing/changing the `distance_band_weights` tests when I have some time.

Reply:
OK sure fine, just wanted to flag :)